Makefile

```diff
@@ -1,3 +1,4 @@
 .PHONY: generate
 generate:
 	$(MAKE) -C resources/mixins/kubernetes generate
+	$(MAKE) -C resources/grafana generate
```

Comment:
Hmm this will currently also update the Kubernetes mixin revision if one just wants to generate the dashboard. Although I guess the problem is more with the mixin generate command, we should separate it from the update part. But we can do that in a separate PR.